src/docs/development/packages-and-plugins/developing-packages.md

```diff
@@ -342,6 +347,19 @@ This is done using a [platform channel][],
 or through the interfaces defined in a platform
 interface package.
 
+### Add support to new platforms in an existing plugin project
```

Comment:
=> Add support for platforms in an existing plugin project
OR
Add platform support in an existing plugin project

Reply:
done